Test/UnitTest1.cs

```diff
@@ -48,13 +44,8 @@ public class UnitTest1 {
    // Test 5: The count of special character in the input array should be equal to the count in the output array
    public void TestMethod5 () {
       var t25 = new T25 ();
-      List<char> input = new ();
-      Random random = new Random ();
-      for (int i = 0; i < 10; i++) {
-         int r = random.Next (0, 26);
-         input.Add ((char)(i % 2 == 0 ? 'a' + r : 'A' + r));
-      }
-      char specialChar = input[random.Next (0, 10)];
+      char[] input = new char[] { 't', 'r', 'U', 'm', 'P', 'M', 'E', 't', 'a', 'm', 'A', 'T', 'I', 'o', 'N' };
+      char specialChar = 'T';
       var output = T25.SpecialSorted (input.ToArray (), specialChar);
       Assert.AreEqual (input.Count (c => c == char.ToLower(specialChar) || c == char.ToUpper (specialChar)), 

```

Comment:
You can just use 'ToUpper' on both input and specialChar

Reply:
How can we use ToUpper () to an array?  There is no ToUpper method for array, right?

Reply:
> How can we use ToUpper () to input, there is no ToUpper method for arrays, right?

Two options, use 'string' which has 'ToUpper' instead of char[], other option is to use 'Select' LINQ operator